src/main/java/com/github/manolo8/darkbot/core/api/GameAPIImpl.java

```diff
@@ -447,18 +447,26 @@ public void resetCache() {
 
     @Override
     public void handleRefresh() {
+        handleRefresh(false);
+    }
+
+    @Override
+    public void handleRefresh(boolean useFakeDailyLogin) {
         // No login has happened? Make a first attempt
         if (hasCapability(Capability.LOGIN) && loginData.getUrl() == null) {
             handleRelogin();
         }
 
         setData(); //always set data to update possible settings changes
         refreshCount++;
-        handler.reload();
-
+        reload(useFakeDailyLogin);
         extraMemoryReader.resetCache();
     }
 
+    protected void reload(boolean useFakeDailyLogin) {
+        handler.reload();
+    }
+
```

Comment:
Why create a new function that does the same as the old one?
If you do not use the boolean of useFakeDailyLogin you do not need a new function asking for it.

Reply:
It's default implementation, as you see I further override this method in KekkaPlayerAdapter